doc/tutorials/enable_ptm.rst

```diff
@@ -0,0 +1,67 @@
+.. _enable-ptm:
+
+Enable PCIe Precision Time Management
+#####################################
+
+The PCI Express (PCIe) specification defines a Precision Time Measurement (PTM)
+mechanism that lets you coordinate and synchronize events across multiple PCI
+components within the same system with very fine time precision.
+
+ACRN adds PCIe root port emulation in the hypervisor to support the PTM feature
+and emulates a simple PTM hierarchy.  ACRN enables PTM in a Guest VM if the user
+sets the ``enable_ptm`` option when passing through a device to a post-launched
+VM. When the user enables PTM, the passthru device is connected to a virtual
+root port instead of the host bridge as it normally would.
+
+Here is an example launch script that enables PTM on a passthru Ethernet card:
+
+.. code-block:: bash
+   :emphasize-lines: 5
+
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+      -s 3,virtio-blk,uos-test.img \
+      -s 4,virtio-net,tap0 \
+      -s 5,virtio-console,@stdio:stdio_port \
+      -s 6,passthru,a9/00/0,enable_ptm \
+      --ovmf /usr/share/acrn/bios/OVMF.fd
+
+Here is the bus hierarchy from guest (as shown by the ``lspci`` command)::
+
+   sudo lspci -tv
+   -[0000:00]-+-00.0  Network Appliance Corporation Device 1275
+              +-03.0  Red Hat, Inc. Virtio block device
+              +-04.0  Red Hat, Inc. Virtio network device
+              +-05.0  Red Hat, Inc. Virtio console
+              \-06.0-[01]----00.0  Intel Corporation Device 15f2
```

Comment:
It looks like the `pci.ids` does not have information about the Ethernet controller. Do we need to update it?

Reply:
if you use "sudo lspci" it will output the information about the ethernet controller.  I think this is how "-t" option of lspci command format its output.